generators/ansible-do/templates/ansible/stack/hosts

```diff
@@ -11,6 +11,9 @@ swarm_workers
 [portainer]
 leader
 
+[postgres]
```

Comment:
This only applies when postgres data folders are installed on the leader node (ie NOT using nfs, and also assuming postgres will be installed on the leader node).